packages/prisma-client/migrations-cli/umzug.ts

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable no-console */
+
+import { Umzug, FileLocker, memoryStorage } from "umzug";
+import fs from "node:fs";
+import path from "node:path";
+import child_process from "node:child_process";
+import { prisma } from "../src";
+
+const runPrismaCLI = (args: string[]) => {
+  console.info(`Running: prisma ${args.join(" ")}\n`);
+
+  const bin = path.resolve("./node_modules/.bin/prisma");
+  if (fs.existsSync(bin) === false) {
+    console.error(
+      "Could not find Prisma CLI binary. Running `yarn install` might help.\n"
+    );
+    process.exit(1);
+  }
+  try {
+    child_process.execFileSync(bin, args, {
```

Comment:
have you seen shelljs? https://www.npmjs.com/package/shelljs 

Would it be worth it? this code is not hard, but there might be edge cases I am not aware off, just as an idea in case we can't use prisma API directly, because wrapping the shell script is definitely less optimal

Reply:
I thought about using some package, but still on the fence about it

Reply:
Using `execa` now